buildlib/tools/zombie_killer.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eEx
+
+# Search for zombie or defunct processes; kill them all
+out=$(ps -eo pid,stat | awk '$2 == "Z|defunct" {print $1}')
+if [ -z "$out" ]; then
+  echo "No zombies found"
+  exit 0
+else
+  for i in $out ; do
+    kill -9 "$i"
+  done
+fi
+
+# Search again; tell parents you killed their kids; kill parents
+out=$(ps -eo pid,stat | awk '$2 == "Z|defunct" {print $1}')
+for i in $out ; do
+  pid_parent=$(ps -o ppid=$i)
+  kill -s SIGCHLD $pid_parent
+  kill -9 $pid_parent
+done
+
+# DISABLED as it interrupts jobs.
+# Search for processes running for more than 6 hours; kill them all
+# out=$(ps -eo pid,user,etimes | grep 'swx-azu' | awk '$3 > 21600 {print $1}')
```

Comment:
maybe some parameters (username, duration) should come as command line arguments for this script?

Reply:
1. The script is not intended to run interactively.
2. For now, I see no reason to kill long-lasting processes. Keeping the code block commented out for potential future reconsideration.

Reply:
@Alexey-Rivkin maybe better remove this commented out code? can always keep it somewhere else
we prefer to not have commented out code in upstream